diff --git a/configure.ac b/configure.ac
index 2cd0b21..b7b40ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -310,0 +311,7 @@ case "$target_cpu" in
+   aarch64 | arm64)
+	MACHTYPE=aarch64
+	P_64BIT=1
+        LIB64=1
+	PBYTE_ORDER=PLITTLE_ENDIAN
+	;;
+
diff --git a/configure b/configure
index 560bcbe..350cf49 100755
--- a/configure
+++ b/configure
@@ -4680,0 +4681,7 @@ case "$target_cpu" in
+   aarch64 | arm64)
+	MACHTYPE=aarch64
+	P_64BIT=1
+        LIB64=1
+	PBYTE_ORDER=PLITTLE_ENDIAN
+	;;
+
